### PR TITLE
BL-14699 update node version to nodejs18.x

### PR DESCRIPTION
**Pull request by faggc**
## Description

Include a summary of the change here.

Related issue: [BL-14699](https://dvsa.atlassian.net/browse/BL-14699)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,6 +54,6 @@
     "webpack-merge": "^5.8.0"
   },
   "engines": {
-    "node": "^14.17.3"
+    "node": "^18.16.0"
   }
 }
```
Comment (caoimheb) [inline package.json:59]: CI github actions workflow still looks to be using Node 14. Update there too?

Comment (DariuszJerzewski) [reply package.json:59]: @caoimheb I can't see CI workflow in this repo 🤔

**Review by DariuszJerzewski — APPROVED** [review]
There is an unanswered comment, but looks good to me otherwise.

File: package.json (hunk 1 above)
Comment (caoimheb) [reply package.json:59]: Ah, file is test.yml, was showing as "CI" in the checks. Updates look good 👍

**Review by caoimheb — APPROVED** [review]

**Review by jennifer-choo — APPROVED** [review]

**Pull request merged**